### DISCUSSION TITLE
Query regarding -sel flag in mdvwhole tool:

### DISCUSSION THREAD

**Discussion by Kankanab** [BartBruininks/mdvwhole#45] [Q&A] [answered]
Dear Sir,

I want to make whole a trimeric protein complex as the one monomeric unit of trimer is found to be broken as its periodic image is coming. So, I want to apply PBC correction and want to make it whole. But, I want to know how to select so that the PBC correction can reconstruct the trimeric state ? How can I use -sel flag for that ? Will the algorithm automastically that part and reconstruct the trimeric state if I mention only: mdvwhole -f npt.tpr -x npt.xtc -mol True -o whole.xtc --resolution -1 -wa True.

It would be really great, if you kindly advise me regarding this.

**Comment by BartBruininks** [comment, marked as answer]
As discussed over the email. The line you are using is almost perfect. You need to specify the selection which is described in the readme of this github. It uses the MDAnalysis selection syntax (you can read their documentation). For your system I tried the following which seems to work for making the protein complexes whole.

```
-sel 'name BB SC1 SC2 SC3 SC4 SC5 SC6; not (resname W WF ION or name BB SC1 SC2 SC3 SC4 SC5) '
```

This makes two selections for the densities for MDVWhole to try to make whole. The protein (the first part of the selection up to the semicolon, and the second part describing all that is not CG Martini Solvent or protein). In your case the second selection is the bilayer. Howver, if one of these selections spans the box in a periodic manner in a certain dimension it is impossible to make such a selection more  whole. You could try a slightly lower resoltuion (0.7 or 0.8) to see if that helps but there is a limit due to the coarseness of the (CG) model. If you include all the embrane components which are bound to the protein you will see that you create a network which spans the PBC and therefore cannot be made whole.

You could try to make separate selections for each trimer (each separated by a semicolon) to treat each trimer as an individual density that you try to make whole.

Cheers,

Bart
